src/rules/function-name-case/README.md

```diff
@@ -97,3 +97,125 @@ a {
   background: -WEBKIT-RADIAL-GRADIENT(red, green, blue);
 }
 ```
+
+## Optional options
+
+### `ignoreFunctions: ["/regex/", "non-regex"]`
+
+Ignore case of function names.
+
+For example, with `"lower"`, given:
+
+```js
+["SOME-FUNCTION", "/^get.*$/"]
+```
+
+The following patterns are considered warnings:
+
+```css
+a {
+  color: sOmE-FuNcTiOn();
+}
+```
+
+```css
+a {
+  color: OTHER-SOME-FUNCTION();
+}
+```
+
+```css
+a {
+  color: GetColor();
+}
+```
+
+```css
+a {
+  color: GET_COLOR();
+}
+```
+
+The following patterns are *not* considered warnings:
+
+```css
+a {
+  display: some-function();
```

Comment:
So strings are not case sensitive but regexes are, right? We should make sure to document that. I sort of think strings should be case-sensitive, too, since the only reason I'd imagine you'd end up with both upper and lowercase versions is a typo.